src/Splitio/Domain/CacheAdapterConfigurationOptions.cs

```diff
@@ -15,6 +16,8 @@ public class CacheAdapterConfigurationOptions
         public string UserPrefix { get; set; }
         public TlsConfig TlsConfig { get; set; }
         public int? PoolSize { get; set; }
+        public List<string> ClusterNodes { get; set; }
```

Comment:
I suggest to create `ClusterNodes` class with `endpoints` and `keyHastTag` properties.

Reply:
will do